docs/pages/guides/mesheryctl.md

```diff
@@ -13,6 +13,7 @@ type: guide
 |                   | version             | Displays the version of the Meshery Client (`mesheryctl`) and the SHA of the release binary. | `mesheryctl system version` |
 |                   | help                | Displays help about any command. | `mesheryctl --help` |
 |                   | --mesheryctl-config | (optional) path to Meshery Client (`mesheryctl`) configuration file (`~/.meshery/mesheryctl.yaml`) to overrides defaults. | `mesheryctl perf <args> --mesheryctl-config=~/.meshery/mesheryctl.yaml` |
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE/EKS). | `./meshery  system config gke --token "PATH TO TOKEN"` |
```

Comment:
@chandrashritii you may want to update `./meshery  system config gke --token "PATH TO TOKEN"` to remove `./` and add `ctl`, like so: `mesheryctl  system config gke --token "PATH TO TOKEN"`

Reply:
@leecalcote The change has been made in the latest commit